BO4E/BO/Marktlokation.cs

```diff
@@ -244,14 +244,25 @@ public class Marktlokation : BusinessObject
         [NonOfficial(NonOfficialCategory.CUSTOMER_REQUIREMENTS)]
         public List<Zaehlwerk>? Zaehlwerke { get; set; }
 
+        /// <summary>
+        ///     Liste für Zählwerke der beteiligten Martrolle
+        /// </summary>
+        [JsonProperty(Required = Required.Default, Order = 32, PropertyName = "zaehlwerkeBeteiligteMarktrolle")]
+        [JsonPropertyName("zaehlwerkeBeteiligteMarktrolle")]
+        [ProtoMember(1026)]
+        [JsonPropertyOrder(32)]
+        [NonOfficial(NonOfficialCategory.CUSTOMER_REQUIREMENTS)]
+        public List<Zaehlwerk>? ZaehlwerkeBeteiligteMarktrolle { get; set; }
```

Comment:
mir ist nicht ganz klar, in wie fern sich das hier von der liste von zaehlwerken direkt obendrüber unterscheidet

Reply:
Es soll die Zählwerke der beteiligten Marktlokation enthalten. Und nicht die Zählwerke der Marktlokation.

Reply:
der beteiligten Marktrolle ;), aber sonst ja